serverless/eventing/brokers/serverless-using-brokers.adoc

```diff
@@ -23,12 +23,12 @@ include::modules/serverless-creating-broker-admin-web-console.adoc[leveloffset=+
 
 [id="next-steps_serverless-using-brokers"]
 == Next steps
-* Configure event delivery parameters that are applied in cases where an event fails to be delivered to an event sink. See xref:../../../serverless/eventing/brokers/serverless-event-delivery.adoc#serverless-configuring-event-delivery-examples_serverless-event-delivery[Examples of configuring event delivery parameters].
+* Configure event delivery parameters that are applied in cases where an event fails to be delivered to an event sink. See xref:../../../serverless/eventing/event-delivery/serverless-event-delivery-parameters-config.adoc#serverless-configuring-event-delivery-examples_serverless-event-delivery[Examples of configuring event delivery parameters].
 
 [id="additional-resources_serverless-using-brokers"]
 [role="_additional-resources"]
 == Additional resources
 * xref:../../../serverless/eventing/brokers/serverless-global-config-broker-class-default.adoc#serverless-global-config-broker-class-default[Configuring the default broker class]
 * xref:../../../serverless/eventing/triggers/serverless-triggers.adoc#serverless-triggers[Triggers]
 xref:../../../serverless/eventing/event-sources/knative-event-sources.adoc#knative-event-sources[Event sources]
-* xref:../../../serverless/eventing/brokers/serverless-event-delivery.adoc#serverless-event-delivery[Event delivery]
+* xref:../../../serverless/eventing/event-delivery/serverless-event-delivery-parameters-config.adoc#serverless-event-delivery[Event delivery]
```

Comment:
```suggestion
* xref:../../../serverless/eventing/event-delivery/serverless-event-delivery-parameters-config.adoc#serverless-event-delivery-parameters-config[Event delivery]
```
ID needs to be updated I think because it was changed and the filename was changed